qcodes/tests/test_config.py

```diff
@@ -135,46 +136,55 @@
                   Config.schema_default_file_name: SCHEMA,
                   }
 
+
 @contextmanager
-def default_config():
+def default_config(user_config: Optional[str]=None):
     """
     Context manager to temporarily establish default config settings.
     This is achieved by overwritting the config paths of the user-,
     environment-, and current directory-config files with the path of the
     config file in the qcodes repository.
```

Comment:
do empty strings assigned below to all the file name attributes mean the same as "the path of the config file in the qcodes repository"? I am not sure i'm following what's going on here but at least either the code change is too much or the docstring needs to be adjusted as well.

Reply:
Maybe these can be `None` but it is all good as long as it is not a filename. I chose the empty string to be on the safe side with typing.

Reply:
ok, implementation is fine. My worry is about consistency of the docstring with the code. Now it claims smth that is not happening in the code, namely "overwritting the config paths of the user-, environment-, and current directory-config files __with the path of the config file in the qcodes repository__". What's happening now is "overwritting the config paths of the user-, environment-, and current directory-config files __with empty strings__", right?